.github/workflows/validatePullRequest.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.x
```

Comment:
In my experience, using the `8.x` SDK fails to build the net6.0 maui targets even with the workloads installed. So, we can maybe revert the changes here for now and look into a major version release?

Reply:
after giving it more thoughts, I believe you are correct. Given that:
- the package target framework include some version specific TFMs (e.g. net6.0-android)
- the workload install command will only obey to the current framework version and the one in global.json (no more way to say "I really want the workload, but for this version instead")
- some workloads do not exist anymore with net8 because they've been integrated directly (maui), which means a tentative to install the given workload under net8 will result in "no workload found"
- changing the TFMs would be a source breaking change.

We'll only be able to remove net6 in the CI when the net6 specific TFMs are updated/removed. Which would require a major version bump.
At the very least, that's until they [fix workload installation with net8](https://github.com/dotnet/sdk/issues/35455) (right now I'm stuck locally trying to build where workload restore tells me everything is here, but build tells me things are missing)